project.json

```diff
@@ -0,0 +1,3 @@
+{
+    "version": "0.0.19"
```

Comment:
I like to revert to v0.0.1.

Reply:
Of course!

Reply:
The Conventional Release Action automatically determines the next version number for you, so I resetted it to `v0.0.0`.